draft-ietf-quic-transport.md

```diff
@@ -697,6 +697,16 @@ the cryptographic handshake provides QUIC with:
   client can receive packets that are addressed with the transport address that
   is claimed by the client (see {{source-address-token}})
 
+The initial cryptographic handshake message needs to be sent in a single packet.
+This avoids having to reassemble a message from multiple packets. This would
+require that servers maintain state prior to establishing a connection, exposing
+servers to a denial of service risk.
+
+The first client packet of the cryptographic handshake protocol MUST fit within
+a 1280 octet QUIC packet.  this includes overheads that reduce the space
```

Comment:
Well, we can't exclude implementations that choose to only send 1280 octet packets.  That would be unwise, I think.  This is MORE strict, I agree, but I think that we want to specifically exclude handshake packets that exceed this or we will find that they aren't properly applicable across multiple networks and use cases.  

If a handshake packet grows too much, then we limit this to networks that can handle a larger MTU.  Given that we don't need the space right now, this seems like a constraint that won't be hard to meet.